nano/node/wallet.cpp

```diff
@@ -1633,17 +1633,17 @@ void nano::wallets::reload ()
 	}
 	// Delete non existing wallets from memory
 	std::vector<nano::wallet_id> deleted_items;
-	for (auto i : items)
```

Comment:
I changed the name of `i` here because it was overshadowing the previously declared `i`